[2025-08-19T10:40:58.541+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-19T10:40:58.581+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_audit_dag.final_status_task manual__2025-08-19T10:40:14.119654+00:00 [queued]>
[2025-08-19T10:40:58.598+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_audit_dag.final_status_task manual__2025-08-19T10:40:14.119654+00:00 [queued]>
[2025-08-19T10:40:58.600+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-08-19T10:40:58.623+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): final_status_task> on 2025-08-19 10:40:14.119654+00:00
[2025-08-19T10:40:58.642+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=728) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-19T10:40:58.647+0000] {standard_task_runner.py:63} INFO - Started process 732 to run task
[2025-08-19T10:40:58.645+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_audit_dag', 'final_status_task', 'manual__2025-08-19T10:40:14.119654+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/Assessment_2/data_audit_dag.py', '--cfg-path', '/tmp/tmpbwass5gd']
[2025-08-19T10:40:58.652+0000] {standard_task_runner.py:91} INFO - Job 54: Subtask final_status_task
[2025-08-19T10:40:58.791+0000] {task_command.py:426} INFO - Running <TaskInstance: data_audit_dag.final_status_task manual__2025-08-19T10:40:14.119654+00:00 [running]> on host 37fc8edea13d
[2025-08-19T10:40:58.942+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='youremail@example.com' AIRFLOW_CTX_DAG_OWNER='rajpriya' AIRFLOW_CTX_DAG_ID='data_audit_dag' AIRFLOW_CTX_TASK_ID='final_status_task' AIRFLOW_CTX_EXECUTION_DATE='2025-08-19T10:40:14.119654+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-19T10:40:14.119654+00:00'
[2025-08-19T10:40:58.946+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-19T10:40:58.948+0000] {logging_mixin.py:188} INFO - [Final]  Audit Success for user: 101
[2025-08-19T10:40:58.949+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-19T10:40:58.951+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-19T10:40:58.968+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_audit_dag, task_id=final_status_task, run_id=manual__2025-08-19T10:40:14.119654+00:00, execution_date=20250819T104014, start_date=20250819T104058, end_date=20250819T104058
[2025-08-19T10:40:59.028+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-19T10:40:59.073+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-19T10:40:59.078+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
